### ISSUE TITLE
Evaluation Protocol

### ISSUE THREAD

**Issue by woojung-son** [lixiaotong97/DSU#4]
Thanks for your awesome research!
I have some questions about the experimental settings.

Q1. Could I ask the evaluation protocol you use? Did you split the images from training domains to train : val when you search hyper-parameters?
Q2. Did you also train the data in validation set when you report the final result?

**Comment by lixiaotong97** [comment]
Hi, thanks for your question. As mentioned by previous papers in DG, when the testing domain is divergent to training domains, the training validation set is not reliable. Thus we follow them to adopt the last checkpoint evaluation protocal. 
Besides, we do not change the training data splits and maintain the same as their original repo respectively.
